src/ui/components/common/sidebar/user_stat_row.rs

```diff
@@ -0,0 +1,12 @@
+use leptos::*;
+use pretty_num::PrettyNumber;
+
+#[component]
+pub fn UserStatRow(count: i64, text: Signal<String>) -> impl IntoView {
```

Comment:
`Signal` is also read-only. Since this component is only meant to be used in the context of the sidebar and it always uses the return value of `move_tr!`, which is `Signal<String>`, I think it's a bit simpler to just use `Signal` here.

Also worth noting that `ReadSignal`s have no way of creating one directly like can be done with `Signal::derive`; rather, they are returned from either `create_signal` (as the first value in the tuple it returns), `RwSignal::read_only`, or `RwSignal::split` (again, as the first value in the return tuple similar to `create_signal`.

Reply:
Ah, so move_tr requires it, gotcha.